mtcpclient/reporting.go

```diff
@@ -1,37 +1,38 @@
 package mtcpclient
 
 import (
-	"github.com/dachad/check-max-tcp-connections/tcpclient"
 	"fmt"
+	"github.com/dachad/check-max-tcp-connections/tcpclient"
 	"time"
 )
 
 func collectConnectionsStatus(connectionDescriptions []tcpclient.Connection, statusChannel <-chan tcpclient.Connection) {
 	for {
 		connectionStatus := <-statusChannel
 		connectionDescriptions[connectionStatus.Id] = connectionStatus
-
 	}
 }
 
-func reportConnectionsStatus(connectionDescriptions []tcpclient.Connection, intervalBetweenUpdates int) {
+func ReportConnectionsStatus(connectionDescriptions []tcpclient.Connection, intervalBetweenUpdates int) {
 	for {
 		fmt.Println(tcpclient.PrintGroupOfConnections(connectionDescriptions))
+		if intervalBetweenUpdates == 0 {
+			break
+		}
 		time.Sleep(time.Duration(intervalBetweenUpdates) * time.Second)
 	}
 }
 
-
-// StartReportingLogic starts some goroutines to capture and report data from the tcpclient
+// startReportingLogic starts some goroutines (so it's not blocking) to capture and report data from the tcpclient
```

Comment:
use `StartReportingLogic` to be consistent with method definition

Reply:
Renames in the IDE do not go trough comments :(